scripts/package/macos-package-app.sh

```diff
@@ -41,21 +42,29 @@ if [ -f scripts/package/appledev.p12 ] && [ -n "$appledev_p12_password" ]; then
     CODESIGN=1
 fi
 
+# Submit app for notarization on macOS >= 10.14
+if [ $MACOS_VERSION_MAJOR -eq 10 ] && [ $MACOS_VERSION_MINOR -ge 14 ]; then
```

Comment:
Is it intentional it doesn't match macos >= 11.0 ?

Reply:
Yes, see discussion with samj and mineo above. Basically there is no real need to complicate the check here.